admin_guide/sdn_troubleshooting.adoc

```diff
@@ -398,21 +398,30 @@ default via 192.168.122.1 dev ens3
 
 You should see the 10.128.x.x lines (assuming you have your pod network set to the default range in your configuration). If you do not, check the {product-title} logs (see the <<reading-the-logs>> section)
 
-=== Is the Open vSwitch configured correctly?
+=== Is the Open vSwitch (OVS) configured correctly?
 
-Check the Open vSwitch bridges on both sides:
+You must run the `ovs-vsctl` and `ovs-ofctl` commands on one of the OVS pods.
+
+To list the OVS pods, enter the following command:
+
+----
+$ oc get pod -n openshift-sdn -l app=ovs
+----
+
+Check the Open vSwitch bridges on both sides.
+Replace `<ovs_pod_name>` with the name of one of the OVS pods.
 
 ----
-# ovs-vsctl list-br
+$ oc exec -n openshift-sdn <ovs_pod_name> -- ovs-vsctl list-br
 br0
 ----
 
-This should be `br0`.
+The previous command should return `br0`.
```

Comment:
What if it's not? Is this not clear enough from the command output?

Reply:
That should never happen, it would need some more investigation to determine why. I don't think we can't give a generic answer to that because it wouldn't be a well-known issue.

If there are more switches present something or someone is creating them, and if br0 isn't present there is a problem with the SDN. Either way it would need further investigation.